release-notes/release-notes/5.7-release-notes.md

```diff
@@ -220,15 +220,19 @@ This links to the Execution Log table with a filter set for your specific sync,
 
 We now log the full REST Target HTTP response in the data sync Execution Errors table to provide you with more detailed information about your job. This replaces the original log that only contained the HTTP response status code.
 
+**CIN-05737 Keep the action bar visible at all times**
```

Comment:
@cysonya here is relevant release note. Sorry for all the noise. Should have saved other changes for release-candidate branch.